Subject: [PATCH] add rules for the symlink of /var/log - audisp_remote_t

We have added rules for the symlink of /var/log in logging.if,
while audisp_remote_t uses /var/log but does not use the
interfaces in logging.if. So still need add a individual rule for
audisp_remote_t.

Upstream-Status: Inappropriate [only for Poky]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
---
 policy/modules/system/logging.te |    1 +
 1 file changed, 1 insertion(+)

--- a/policy/modules/system/logging.te
+++ b/policy/modules/system/logging.te
@@ -280,10 +280,11 @@ optional_policy(`
 
 allow audisp_remote_t self:capability { setpcap setuid };
 allow audisp_remote_t self:process { getcap setcap };
 allow audisp_remote_t self:tcp_socket create_socket_perms;
 allow audisp_remote_t var_log_t:dir search_dir_perms;
+allow audisp_remote_t var_log_t:lnk_file read_lnk_file_perms;
 
 manage_dirs_pattern(audisp_remote_t, audit_spool_t, audit_spool_t)
 manage_files_pattern(audisp_remote_t, audit_spool_t, audit_spool_t)
 files_spool_filetrans(audisp_remote_t, audit_spool_t, { dir file })
 
